database/dbase.go

```diff
@@ -280,7 +280,7 @@ func (rcdb *ReconnectableCloudtrustDB) checkError(err error) {
 			return
 		}
 		if connection.Ping() != nil {
-			rcdb.resetConnection(true)
+			_ = rcdb.resetConnection(true)
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time